doc/content/getting-started/guides/build-an-end-to-end-solution-gn/_index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Building an End-to-End LoRaWAN Application with The Things Stack"
+description: ""
+weight: 
+---
+
+In this section, we will show how to build an end-to-end solution with [{{% tts %}}](/getting-started/what-is-tts/) to control the [Kuando Busylight](https://busylight.com/busylight-iot/) with the [Generic Node Sensor Edition](https://www.thethingsshop.com/products/generic-node-sensor-edition)(GNSE). 
+
+# Registering The Things Indoor Gateway
+
+Register The Things Indoor Gateway with {{% tts %}} by following the instructions on this [page](/gateways/models/thethingsindoorgateway/). Once registered, your gateway will be ready to receive uplinks and forward downlinks.
```

Comment:
```suggestion
Register The Things Indoor Gateway with {{% tts %}} by following [these instructions]({{< "/gateways/models/thethingsindoorgateway" >}}). Once registered, your gateway will be ready to receive uplinks and forward downlinks.
```

Reply:
Are you sure that putting the link without {{< >}} works in production? I'm not sure